release/_bom.sh

```diff
@@ -76,6 +76,37 @@ function generate_api_jars {
 	do
 		_manage_bom_jar "${module_jar}"
 	done
+
+	for file in $(ls api-jar/META-INF --almost-all | grep --extended-regexp --invert-match '^(alloy-util.tld|alloy.tld|c.tld|liferay.tld)$')
+	do
+		if [[ "$file" == *.tld ]]
+		then
+			rm "api-jar/META-INF/${file}"
+		fi
+	done
+
+	local tld_files=($(find "${_PROJECTS_DIR}" -name "liferay-*.tld" -o -name "ratings.tld" -type f | awk -F/ '{print $NF, $0}' | sort -k1,1 -u | awk '{print $2}'))
```

Comment:
@lucasmiranda0, can we change to `awk -F /`? I mean, add a blank space between `-F` and `/`

In addition, can you please do the following?

```
sort -k 1,1 -u
```